### DISCUSSION TITLE
Restoring a LinkAce Backup

### DISCUSSION THREAD

**Discussion by mztiq** [Kovah/LinkAce#446] [Support]
Hi there,

I finally managed to get the backup to local disc working, and now I'm wondering how I'd restore these backups in case I need to.
I'm not really familiar with the PHP artisan commands, I tried to find a suitable command  without success.
A hint into the right direction would be appreciated.

**Comment by Kovah** [comment]
Technically, you only need to restore the .env file and the database. LinkAce does not store any other data that is crucial.

**Comment by mztiq** [comment]
Thanks for the quick response and sorry for the noobish question but how exactly can I restore the database?
I have a backup of LinkAce which is a .zip file containing an "app" folder and a "db-dumps" folder, inside the "db-dumps" folder there is a "mysql-linkace.sql" but I'm not sure how I would restore this.

I couldn't find anything like a "mysql-linkace.sql" in the linkace_db docker volume.

**Reply by Kovah** [reply]
You can try something like this, adopted to your database credentials:
```
docker exec -i some-mariadb sh -c 'exec mysql -uroot -p"$MARIADB_ROOT_PASSWORD"' < /some/path/on/your/host/all-databases.sql
```
Taken from https://registry.hub.docker.com/_/mariadb
